db/schema.rb

```diff
@@ -98,9 +98,9 @@
     t.datetime "last_session"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "recordings_processing", default: 0
     t.string "viewer_access_code"
     t.string "moderator_access_code"
-    t.integer "recordings_processing", default: 0
```

Comment:
It's better if we leave the schema for migrations to shape or we'll be having a lot of changes whenever a migration change occurs.